simple-facebook-instant-articles.php

```diff
@@ -178,21 +179,20 @@ public function setup_content_mods() {
 		add_shortcode( 'sigallery', array( $this, 'api_galleries_shortcode' ) );
 
 		// Render social embeds into FB IA format.
-		add_filter( 'embed_handler_html', array( $this, 'fb_formatted_social_embeds' ), 10, 3 );
-		add_filter( 'embed_oembed_html', array( $this, 'fb_formatted_social_embeds' ), 10, 4 );
-
-		// Render post content via DOM - to format it into FB IA format.
-		// DO it last, so content was altered via WP native hooks as much as possible.
-		add_filter( 'the_content', array( $this, 'fb_formatted_post_content' ), 1000 );
+		add_filter( 'embed_handler_html', array( $this, 'get_social_embed' ), 10, 3 );
+		add_filter( 'embed_oembed_html', array( $this, 'get_social_embed' ), 10, 4 );
 
-		// GA analytics code.
+		// Modify the content
+		add_filter( 'the_content', array( $this, 'reformat_post_content' ), 1000 );
 		add_action( 'the_content', array( $this, 'append_google_analytics_code' ), 1100 );
+		add_action( 'the_content', array( $this, 'append_ad_code' ), 1100 );
 
 		// Post URL for the feed.
 		add_filter( 'the_permalink_rss', array( $this, 'rss_permalink' ) );
 
 		// Render post content into FB IA format - using DOM object.
-		add_action( 'simple_fb_formatted_post_content', array( $this, 'render_pull_quotes' ), 10, 2 );
+		add_action( 'simple_facebook_ia_reformat_post_content', array( $this, 'render_pull_quotes' ), 10, 2 );
```

Comment:
Should we rename this to `simple_facebook_ia_reformat_post_content` to keep consistent with other hook names using `simple_fb_***`?